### DISCUSSION TITLE
unfortunate mixing of inspire theme vocabulary locations in TG example

### DISCUSSION THREAD

**Discussion by pvgenuchten** [INSPIRE-MIF/helpdesk#129] [Q&A] [answered]
From metadata repository:

In this TG example there is an unfortunate mixing of 2 available locations of the inspire themes thesaurus

![image](https://user-images.githubusercontent.com/299829/65784780-20c7a200-e153-11e9-9fa6-acea3c77b16d.png)

usage of INSPIRE registry URI's for concepts
usage of EIONET URI's for the thesaurus

I noticed some in our domain considering the combination of these 2 locations as a formal recommendation, however combining of the 2 locations should be strongly prevented.

I wonder if we could consider something in the ATS to fix this issue, or should this go back to the TG group?

Why shouldn't there be a combination of the 2 thesauri locations? The 2 thesauri use different URI's for the same concepts. A machine parsing the metadata would expect to find in the metadata the same uri identifying the thesaurus, as well as in the inScheme property of the concept.

```
<skos:inScheme rdf:resource="http://inspire.ec.europa.eu/theme" />
```

Or vice versa, if, in a metadata edit tool, users are enabled to select a keyword from a thesaurus, the thesaurus identifier would typically be filled with the inscheme property of the selected concept.

In case the combination of thesauri locations is actually intentional, then i would like to propose to use in both thesauri locations the same uri's.

How do others perceive this challenge?

**Comment by pvgenuchten** [comment]
We had some discussion on the note under requirement 1.4

![image](https://user-images.githubusercontent.com/299829/66119681-66fc8580-e5d9-11e9-9793-9c383c5ed9de.png)

Does the note imply that uri's from the inspire registry can also be used? Or does TG indicate the use of eionet uri's? 

Considering above, when trying to link to a theme concept in eionet, we noticed there doesn't seem to be a language-neutral uri for a concept. There are uri's in http://rdfdata.eionet.europa.eu/inspirethemes/themes.rdf (http://rdfdata.eionet.europa.eu/inspirethemes/ontology/InspireTheme#1), but these don't resolve to content. We see no other option then to include the language specific uri for a theme: http://www.eionet.europa.eu/gemet/nl/inspire-theme/ps

**Comment by MarcoMinghini** [comment]
This issue was originally opened in the [Validator helpdesk](https://github.com/INSPIRE-MIF/helpdesk-validator/issues/). We moved it here, since it addresses a more general INSPIRE discussion topic and is not Validator-specific.

**Comment by sMorrone** [comment]
@MarcoMinghini @pvgenuchten 
this issue should become a change proposal to MD TG in the[ technical guidance repository issue tracker](https://github.com/INSPIRE-MIF/technical-guidelines/issues) where it will be subject to  specific governance and release process

**Comment by heidivanparys** [comment, marked as answer]
The [IR metadata](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:02008R1205-20081224#tocId24) says:

> If a resource is a spatial data set or spatial data set series, at least one keyword shall be provided from the general environmental multilingual thesaurus (GEMET) describing the relevant spatial data theme as defined in Annex I, II or III to Directive 2007/2/EC.

I assume that is the reason link http://www.eionet.europa.eu/gemet/inspire_themes link is used in the TG (recommendation metadata/2.0/rec/datasets-and-series/use-anchors-for-gemet), and not http://inspire.ec.europa.eu/theme.
